README.md

```diff
@@ -3,54 +3,60 @@
 [![Code Climate](https://codeclimate.com/github/turboMaCk/koa-sslify/badges/gpa.svg)](https://codeclimate.com/github/turboMaCk/koa-sslify)
 [![npm version](https://badge.fury.io/js/koa-sslify.svg)](https://badge.fury.io/js/koa-sslify)
 
-**This is NEXT version of Koa SSLify compatible with Koa@2.0.0! This version is released under 2.0.0 tag.
+**This is the NEXT version of Koa SSLify compatible with Koa@2.0.0! This version is released under the koa2 branch.
 For stable please check [master](https://github.com/turboMaCk/koa-sslify/) branch.**
 
 This simple [Koa.js](http://koajs.com/) middleware enforces HTTPS connections on any incoming requests.
-In case of a non-encrypted HTTP request, koa-sslify automatically redirects to an HTTPS address using a 301 permanent redirect.
+In case of a non-encrypted HTTP request, koa-sslify automatically redirects to an HTTPS address using a `301 permanent redirect`.
 
 koa-sslify also works behind reverse proxies (load balancers) as they are for example used by Heroku and nodejitsu.
-In such cases, however, the trustProxy parameter has to be set (see below).
+In such cases, however, the `trustProxy` parameter has to be set (see below).
 
 ## Install
 ```
-$ npm install koa-sslify@2.0.0
+$ npm install koa-sslify@2.0.1
```

Comment:
I think with API change (non breaking one) next version should be 2.1.0. http://semver.org/
Anyway I can change this by myself.